app/models/proposal.rb

```diff
@@ -0,0 +1,13 @@
+class Proposal < ActiveRecord::Base
+  has_one :cart
```

Comment:
Yep, `has_one` does that, though probably worth noting that it won't always be present in an inline comment. Good call!